shared/constants/types/config.tsx

```diff
@@ -17,5 +16,4 @@ export type ConnectionType = NetInfo.NetInfoStateType | 'notavailable'
 
 export type State = {
   loggedIn: boolean
-  startupFile: HiddenString
```

Comment:
so we set this, then never do anything with it. also seems like a thing no one ever does